lib/thredded.rb

```diff
@@ -56,9 +57,12 @@ def self.user_class
     end
   end
 
-  def self.user_path(user)
+  # @param context [Object] context to execute the lambda in.
+  # @param user [Thredded.user_class]
+  # @return [String] path to the user evaluated in the specified context.
+  def self.user_path(context, user)
```

Comment:
Considering the several (few, but distinct) ways that the term `context` is used in this code base I'm wondering if you would consider a bit of a name shift for "context" when we talk about the view instance we're passing around. When I see "context" in isolation - like, per method - I'm not sure I completely grok what it means. What do you think about `view_instance` or `view_context` or something to that effect?